golang/lib/networks/network_context.go

```diff
@@ -202,75 +195,44 @@ func (networkCtx *NetworkContext) AddServiceToPartition(
 		serviceId,
 		serviceIpAddr,
 		networkCtx.enclaveDataVolMountpoint,
-		containerCreationConfig.GetKurtosisVolumeMountpoint())
+		kurtosisVolumeMountpoint,
+		)
 	logrus.Tracef("New service successfully registered with Kurtosis API")
 
-	logrus.Trace("Loading static files into new service namespace...")
-	usedStaticFiles := containerCreationConfig.GetUsedStaticFiles()
-	staticFileAbsFilepathsOnService, err := serviceContext.LoadStaticFiles(usedStaticFiles)
-	if err != nil {
-		return nil, nil, stacktrace.Propagate(err, "An error occurred loading the following static files to service '%v': %+v", serviceId, usedStaticFiles)
-	}
-	logrus.Trace("Successfully loaded static files")
+	return serviceContext,  nil
+}
 
-	logrus.Trace("Initializing generated files...")
-	filesToGenerate := map[string]bool{}
-	for fileId := range containerCreationConfig.GetFileGeneratingFuncs() {
-		filesToGenerate[fileId] = true
-	}
-	generatedFileFilepaths, err := serviceContext.GenerateFiles(filesToGenerate)
-	if err != nil {
-		return nil, nil, stacktrace.Propagate(err, "An error occurred generating the files needed for service startup")
-	}
-	generatedFileAbsFilepathsOnService := map[string]string{}
-	for fileId, initializingFunc := range containerCreationConfig.GetFileGeneratingFuncs() {
-		filepaths, found := generatedFileFilepaths[fileId]
-		if !found {
-			return nil, nil, stacktrace.NewError(
-				"Needed to initialize file for file ID '%v', but no generated file filepaths were found for that file ID; this is a Kurtosis bug",
-				fileId)
-		}
-		fp, err := os.Create(filepaths.GetAbsoluteFilepathHere())
-		if err != nil {
-			return nil, nil, stacktrace.Propagate(err, "An error occurred opening file pointer for file '%v'", fileId)
-		}
-		if err := initializingFunc(fp); err != nil {
-			return nil, nil, stacktrace.Propagate(err, "The function to initialize file with ID '%v' returned an error", fileId)
-		}
-		generatedFileAbsFilepathsOnService[fileId] = filepaths.GetAbsoluteFilepathOnServiceContainer()
-	}
-	logrus.Trace("Successfully initialized generated files in suite execution volume")
+func (networkCtx *NetworkContext) StartService(
+		serviceContext *services.ServiceContext,
+		containerConfig *services.ContainerConfig) (map[string]*kurtosis_core_rpc_api_bindings.PortBinding, error) {
 
-	containerRunConfig, err := generateRunConfigFunc(serviceIpAddr, generatedFileAbsFilepathsOnService, staticFileAbsFilepathsOnService)
-	if err != nil {
-		return nil, nil, stacktrace.Propagate(err, "An error occurred getting the container run config")
-	}
+	ctx := context.Background()
 
 	logrus.Tracef("Creating files artifact ID str -> mount dirpaths map...")
 	artifactIdStrToMountDirpath := map[string]string{}
-	for filesArtifactId, mountDirpath := range containerCreationConfig.GetFilesArtifactMountpoints() {
+	for filesArtifactId, mountDirpath := range containerConfig.GetFilesArtifactMountpoints() {
```

Comment:
I reallyyyyyy like the simplification of `containerCreation/RunConfig -> containerConfig`

Reply:
tyty :)